modules/token/lib/token-price.service.ts

```diff
@@ -16,13 +16,15 @@ import { LinearWrappedTokenPriceHandlerService } from './token-price-handlers/li
 import { BptPriceHandlerService } from './token-price-handlers/bpt-price-handler.service';
 import { SwapsPriceHandlerService } from './token-price-handlers/swaps-price-handler.service';
 import { PrismaTokenWithTypes } from '../../../prisma/prisma-types';
+import { AavePriceHandlerService } from './token-price-handlers/aave-price-handler.service';
 
 export class TokenPriceService {
     cache: CacheClass<string, any> = new Cache<string, any>();
     private readonly priceHandlers: TokenPriceHandler[] = [
         new FbeetsPriceHandlerService(),
         new ClqdrPriceHandlerService(),
         new CoingeckoPriceHandlerService(),
+        new AavePriceHandlerService(),
```

Comment:
Like this, if these tokens ever get a coingecko feed, they will be priced via coingecko. Is that the intention? I think I would move it above coingecko, as I would assume _if_ coingecko ever adds such a feed, it will be wrong at the beginning.